Sources/SwifterSwift/UIKit/UIViewExtensions.swift

```diff
@@ -203,8 +203,8 @@ public extension UIView {
         }
     }
 
+    // swiftlint:disable identifier_name
```

Comment:
Not in this patch, but I think it would be great add identifiers like: x, y, z, i, j as exceptions for identifier name on swiftlint so we could avoid having to do this all the time for those well-known var names. Maybe we could add an issue with `good first issue` label :))

Reply:
I know it is common, but I didn't add it to `.swiftlint.yml` to still get warnings for newly added extensions